eos_data_distribution/ndn/dbus/base.py

```diff
@@ -251,9 +251,9 @@ def __init__(self, name, dbus_name, skeleton):
         self._dbus_name = build_dbus_name(dbus_name, name)
         self._interface_skeleton_object = skeleton
 
-        address = Gio.dbus_address_get_for_bus_sync(BUS_TYPE, None)
-        self.con = Gio.DBusConnection.new_for_address_sync(address, Gio.DBusConnectionFlags.NONE, None, None)
 
+        address = Gio.dbus_address_get_for_bus_sync(BUS_TYPE, None)
+        self.con = Gio.DBusConnection.new_for_address_sync(address, Gio.DBusConnectionFlags.AUTHENTICATION_CLIENT | Gio.DBusConnectionFlags.MESSAGE_BUS_CONNECTION, None, None)
```

Comment:
from the docs: 
```
The returned object is a singleton, that is, shared with other callers of g_bus_get() and g_bus_get_sync() for bus_type . In the event that you need a private message bus connection, use g_dbus_address_get_for_bus_sync() and g_dbus_connection_new_for_address().
```

in store we have the base connection and we need another connection for the chunks code so that they are properly separated